libs/components/src/tooltip/stories/vl-tooltip.stories-doc.mdx

```diff
@@ -0,0 +1,48 @@
+import { ArgTypes, Canvas } from '@storybook/blocks';
+import * as VlTooltipStories from './vl-tooltip.stories';
+
+# Tooltip
+
+#### ❗DEPRECATED: gebruik de [vl-popover](?path=/docs/components-popover--documentatie) component❗<br/>
+
+<br />
+Gebruik een `tooltip` om beschrijvende informatie te geven over de functie van een knop, label of ander element. Wanneer
```

Comment:
Gebruik de `tooltip` component om beschrijvende informatie over de functie van een knop, label of ander element weer te geven.